### PR TITLE
Feature/142/remove swagger use open api

### PR DESCRIPTION
**Pull request by garyluu**
For #142 

- Removed the swagger yaml file. 
- Disabled the swagger -> openapi conversion
- Simplified some openapi parameters
- Removed some obvious incorrect openapi responses

Note: There are some more obvious incorrect openapi responses (any endpoint that returns an array of files models) but are less easy to fix

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -48,16 +42,13 @@ paths:
...
               schema:
-                $ref: "#/components/schemas/Tool"
+                type: string
         "404":
           description: The tool can not be found.
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-            text/plain:
```
Comment (coverbeck) [inline openapi/openapi.yaml:58]: To confirm, this must be true, but the Dockstore implementation  always returns a JSON for errors?

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -99,41 +79,21 @@ paths:
...
-        - name: version_id
-          in: path
-          required: true
-          description: An identifier of the tool version, scoped to this registry, for
-            example `v1`. We recommend that versions use semantic versioning
-            https://semver.org/spec/v2.0.0.html  (For example, `1.0.0` instead
-            of `develop`)
-          schema:
-            type: string
+        - $ref: "#/components/parameters/id"
```
Comment (coverbeck) [inline openapi/openapi.yaml:82]: 👍 Don't think I knew about this. Avoids a lot of repetition.

**Review by coverbeck — APPROVED** [review]

File: openapi/openapi.yaml (hunk 1 above)
Comment (garyluu) [reply openapi/openapi.yaml:58]: Not sure about how consistent Dockstore is, but I believe the TRS swagger intended it to be a JSON only

File: openapi/README.md [hunk 3]
```diff
@@ -1,7 +0,0 @@
-## Instruction for modifying the TRS
-
-To make changes to the TRS, join the GA4GH organization or ask to join this repository as a collaborator. Then in your branch, modify:
```
Comment (denis-yuen) [inline openapi/README.md:3]: Leave in line 3 and explain this `v1` folder just in case.

**Review by denis-yuen — APPROVED** [review]

File: openapi/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ## Instruction for modifying the TRS
 
 To make changes to the TRS, join the GA4GH organization or ask to join this repository as a collaborator. Then in your branch, modify:
- - The swagger yaml file (`ga4gh-tool-discovery.yaml`) with a Swagger 2 definition of the changes.
-   - This swagger yaml file will be used in the validation server.
- - An OpenAPI yaml file (`openapi.yaml`) with an OpenAPI 3 definition of the changes will be auto-generated by swagger2openapi and checked in by an automated TravisCI build
+ - The openapi.yaml file with an OpenAPI 3 definition of the changes.
+   - This openapi yaml file will be used in the TRS validation server.
 
+The v1 directory should not be modified. It provides backward compattibility support for TRS servers/clients.
```
Comment (denis-yuen) [inline openapi/README.md:7]: compattibility typo

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**